src/server/services/wms/qgswmsparameters.h

```diff
@@ -191,6 +191,15 @@ namespace QgsWms
        */
       bool isValid() const override;
 
+      /**
+       * Converts the parameter into a list of strings and keeps empty parts
+       * Default style value is an empty string
+       * \param delimiter The character used for delimiting
+       * \returns A list of strings
+       * \since QGIS 3.8
```

Comment:
@pblottiere @elpaso @nyalldawson 
Do I have to update \since before merge ?
Do I add a commit to the branch or do I update the commit (commit, rebase, push force) ?